pyscriptjs/src/components/pyrepl.ts

```diff
@@ -191,7 +193,7 @@ export function make_PyRepl(interpreter: InterpreterClient, app: PyScriptApp) {
         // should be the default.
         autogenerateMaybe(): void {
             if (this.hasAttribute('auto-generate')) {
-                const allPyRepls = document.querySelectorAll(`py-repl[root='${this.getAttribute('root')}'][exec-id]`);
+                const allPyRepls = $$(`py-repl[root='${this.getAttribute('root')}'][exec-id]`);
```

Comment:
Do we want to use an explicit `document` argument for the roothere? I know it's the default for `$$`, but for those not as familiar with the basic dev utils, it might help readability.

Reply:
done